src/Banana/Syntax/Parser.hs

```diff
@@ -52,14 +52,20 @@ term = parens parseExpr
       toDouble (Right d) = d
 
 table :: [[Operator Parser (Expr a)]]
-table = [ [binary "*" Mul AssocLeft, binary "/" Div AssocLeft ]
-        , [binary "+" Add AssocLeft, binary "-" Sub AssocLeft ]
+table = [ [binary "*"  Mul AssocLeft,  binary "/"  Div AssocLeft ]
+        , [binary "+"  Add AssocLeft,  binary "-"  Sub AssocLeft ]
+        , [binary "and" And AssocLeft, binary "or" Or  AssocLeft]
+        , [prefix "not" Not]
+        , [binary "<"  Less AssocNone, binary ">"  More AssocNone ]
+        , [binary "<=" LEq  AssocNone,  binary ">=" MEq AssocNone ]
+        , [binary "="  Eq   AssocRight,  binary "/=" NotEq AssocRight ]
```

Comment:
This messes up the order of operations. Please check the [grammar](https://github.com/aabounegm/banana/wiki/Language-Grammar).